### ISSUE TITLE
Feature request: option to display tests passed/failed, not asserts

### ISSUE THREAD

**Issue by nightblade9** [bitwes/Gut#310]
I would like to request an option to configure Gut so it shows number of tests passed/failed instead of number of asserts. (I'm working on a proc-gen game and running tests that check generated levels, which are random in nature, and may end up with a large number of and/or changing number of assets.)

I'd like to see "67 tests passed" instead of "86999 asserts passed"

Screenshot for posterity:

![image](https://user-images.githubusercontent.com/31222662/132775244-98ae1591-9dea-4770-b441-a1663683c975.png)

**Comment by bitwes** [comment]
I think it should show pass/fail counts for tests as well.  Maybe it should only show the assert count since a number of passing or failing asserts doesn't tell you that much.

**Comment by nightblade9** [comment]
Sure, that would be great.

I was digging through some of the Gut issues and saw a comment from you to the effect of: you usually handle this by expecting asserts to check a number of things and do a single assert to the effect of `times_thing_happened == expected_times`. That's cool. That won't work for my case, though, which is essentially "randomly generate levels for 10 seconds and assert a bunch of things to see if anything is broken."

**Comment by bitwes** [comment]
Updated in master, this will be released with 7.3.  The new display looks like this:
```
Totals
Scripts:          40
Passing tests     1082
Failing tests     6
Pending:          4
Asserts:          1531/13
```

**Comment by nightblade9** [comment]
Awesome!

**Comment by bitwes** [comment]
Released in v7.3.0.
